### ISSUE TITLE
 RetinaNet with custom backbone and custom dataset

### ISSUE THREAD

**Issue by adityaroy10** [keras-team/tf-keras#773]
How to form the dataset for training? 

for :
model.fit( X, Y)  #where model is the retinanet object with appropriate backbone using keras_cv.models.RetinaNet()

X would be the array of images.
So for X0 (first image), will Y0 be array of [class, xmin, xmax, ymin, ymax] for all  objects in the image?
What is the correct format?

Right now, I have the annotations saved in [class, xmin, xmax, ymin, ymax]  format in a separate text file for each image

Is there any alternative for such case?

Thank you for any help.

**Comment by tilakrayal** [comment]
@adityaroy10,
RetinaNet, a popular single-stage detector, which is accurate and runs fast. RetinaNet uses a feature pyramid network to efficiently detect objects at multiple scales and introduces a new loss, the Focal loss function, to alleviate the problem of the extreme foreground-background class imbalance. Also there is **COCO2017** dataset which has around 118k images for the training and testing.

https://keras.io/examples/vision/retinanet/
Also there is the data structure of the coco-dataset for the own custom data using TFDS.
https://www.tensorflow.org/datasets/add_dataset#version

Looks like this issue is not related to tf-keras. Could you please try to file an issue on tensorflow [forum](https://discuss.tensorflow.org/) for such type of issues. Thank you!
